mitm.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package forwarder
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"time"
+
+	"github.com/google/martian/v3/mitm"
+	"github.com/saucelabs/forwarder/utils/certutil"
+)
+
+type MITMConfig struct {
+	CACertFile string
+	CAKeyFile  string
+
+	Organization string
+	Validity     time.Duration
+}
+
+func DefaultMITMConfig() *MITMConfig {
+	return &MITMConfig{
+		Organization: "Sauce Labs Inc.",
+		Validity:     24 * time.Hour, //nolint:gomnd // 24 hours is a reasonable default
```

Comment:
Let's add 24 to ignored-numbers in golangci file. As you explain, 24 is a reasonable value and occurs often when it comes to time.

Reply:
OK

Reply:
I'm disabling gomnd